spec/services/atmosphere/billing_service_spec.rb

```diff
@@ -40,7 +45,9 @@
 
     it 'throws exception for malformed appliance' do
       appl = create(:appliance, fund: nil, appliance_set: wf, appliance_type: not_shareable_appl_type, appliance_configuration_instance: config_inst, virtual_machines: [not_shareable_vm1])
-      appl.update_column(:fund_id, nil)
+
+      appl.update_columns(fund_id: nil)
```

Comment:
This is done to force an invalid active record in order to test out the exception mechanism. I agree it's not a particularly useful spec.